docs/commands/block-states.md

```diff
@@ -42,80 +42,75 @@ In Bedrock Edition we used Aux values (also known as Metadata) to define a block
-
-    -   Example: Redstone power 1 to 15
-    -   `["redstone_power"=10]`
-
--   **Boolean** is a programming term which refers to `true/false` values. You can simply understand it as yes or no questions.
-
-    -   Is this piston powered? `yes/no`
-    -   Is this button pressed? `yes/no`
-    -   Is this log stripped? `yes/no`
-    -   `["stripped_bit"=true]`
-
--   **Strings** are unique 'text' inputs. You can simply understand it as multiple choice questions.
-    -   What color is this wool? `"white"`, `"orange"`, `"brown"` etc..
-    -   What wood type is this log? `"spruce"`, `"birch"`, `"acacia"` etc..
-    -   `["wood_type"="spruce"]`
-
+- **Integers** are whole numbers. They are used to define a block from a 'range' of values.
+    - Example: Redstone power 1 to 15
+    - `["redstone_power"=10]`
+
+- **Boolean** is a programming term which refers to `true/false` values. You can understand them like yes/no questions.
+    - Is this piston powered? `yes/no`
+    - Is this button pressed? `yes/no`
+    - Is this log stripped? `yes/no`
+    - Example: `["stripped_bit"=true]`
+
+- **Strings** are unique 'text' inputs. You can understand them like multiple-choice questions.
+    - What color is this wool? `"white"`, `"orange"`, `"brown"`, etc..
+    - What wood type is this log? `"spruce"`, `"birch"`, `"acacia"`, etc..
+    - Example: `["wood_type"="spruce"]`
+ 
+ 
 ## Block States List
 
-A list of all the block states currently available within Bedrock can be found at:
+A list of all the Block States currently available in Bedrock can be found at:
 https://learn.microsoft.com/en-us/minecraft/creator/reference/content/blockreference/examples/blockstateslist
 
-Note: In the site block states may be written as one word but make sure to separate them with underscores `_` when typing in commands.
-
-Example: `buttonPressedBit` → `"button_pressed_bit"`
+- Note: In the site, Block States may be listed in `camelCase`, but in commands, only enter them in `snake_case`
+    - Example: `buttonPressedBit` → `"button_pressed_bit"`
 
 ## Converting Aux Values to Block States
 
-For your convenience; download any of the excel sheet below to find the full list of block IDs, their aux values and equivalent block states in Bedrock. _Shared by kayla@Mojang_
-
-<Button link="https://github.com/BedrockCommands/bedrockcommands.github.io/files/10987839/Aux-Value_to_Block-States_Map.xlsx">
-    Download Sheet 1
-</Button>
-
-Note: the above sheet was quickly generated and contains some minor errors. Boolean values `0` should be replaced with `false` and `1` should be replaced with `true` since the game doesn't recognize the syntax otherwise.
+For your convenience, you may use this [Lookup Table](https://auxval-to-blockstates.netlify.app/) by *@SmokeyStack* to get the up-to-date list of all the Block IDs, their Aux Values and Block State equivalents.
 
-Alternate sheet: _Shared by @ItsRichHeart_
+## Known Issue
```

Comment:
Better use "Known Issues"

Reply:
Since there are two related bug reports mentioned besides the main issue, I think that should be alright.